test/acceptance/todo-cli-test.js

```diff
@@ -160,6 +161,52 @@ describe('todo usage', () => {
       expect(todoStorageDirExists(project.baseDir)).toEqual(true);
     });
 
+    it.only('does not remove todos from another engine', async function () {
```

Comment:
`.only` 🥩 🔪

Reply:
hopefully this would have failed CI, if not can you file an issue?

Reply:
It should have failed linting, as that rule is for sure enabled.